### PR TITLE
feat(SRE-12888): Add in IAM roles for Helix-agent installation

### PR DESCRIPTION
**Pull request by par-texx**
* Add in roles for Helix-agent installation

### CODE REVIEW

**Review by FernandoMiguel — COMMENTED** [review]
pls set tf to version 1.7
`  required_version = ">= 0.14.1"`

**Comment by FernandoMiguel** [comment]
pls add `.terraform-version`

**Comment by FernandoMiguel** [comment]
pls set aws provider version to v5

File: modules/runners/policies-runner.tf [hunk 1]
```diff
@@ -63,4 +63,59 @@ resource "aws_iam_role_policy" "ec2" {
   policy = templatefile("${path.module}/policies/instance-ec2.json", {})
 }
 
+data "aws_iam_policy_document" "helix_agent_policy" {
```
Comment (macneib) [inline modules/runners/policies-runner.tf:66]: usually we do iam policies roles here https://github.com/enverus-cts/cts.terraform/blob/main/github/runners/iam.tf

**Pull request closed without merging**